DirectEmployers/MyJobs
# [PD 3125] add contact display

## What's this PR do?
Adds the contact display when adding or editing a contact for a cbo

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the CBO Application

## How should this be manually tested?
Pull down repository. Go into a current CBO and add an additional contact or edit a contact and you should see the CBO details in the right hand facet side

## Any background context you want to provide?
PR up again to fix the errors that were breaking on QC
